### ISSUE TITLE
Copy-File copies irrelevant files in 3.8.3 version

### ISSUE THREAD

**Issue by pushpa-mamilla** [PSAppDeployToolkit/PSAppDeployToolkit#596]
Copy-File -Path "$dirAppDeployToolkit\*.*" -Destination "$env:Programdata\Uninstalls" -recurse

the above line copies $RecycleBin folder to destination, although i am targeting to copy AppDeployToolkit contents.
is copy-file restricted to only files and supportfiles folders?

**Comment by luki1412** [comment]
$dirAppDeployToolkit is not a valid variable, so it will be $null or in this case - empty string. So you end up with only *.* and you copy files from current directory to destination. If you created $dirAppDeployToolkit yourself, make sure it contains the correct path.

**Comment by mjr4077au** [comment]
I don't believe there's a legitimate, ongoing issue here. If this issue is still active, please let us know.
